ci-operator/config/openshift/openstack-resource-controller/OWNERS

```diff
@@ -0,0 +1,17 @@
+approvers:
+- emilienm
+- gryf
+- mandre
+- maysamacedo
+- mdbooth
+- pierreprinetti
```

Comment:
Do we want to drop `pierreprinetti`/add `dkokkino`? We can also do a bulk update later.

Reply:
I think for now we add this and we'll fix it in the repo, which will be synced by a bot.